Makefile

```diff
@@ -18,6 +18,11 @@ CLOUDCAFE ?= $(shell which cafe-runner)
 
 .PHONY: targets env-precheck
 
+hooks:
+	echo "#!/bin/bash" > .git/hooks/pre-commit
+	echo "python `pwd`/scripts/config_check.py" >> .git/hooks/pre-commit
+	echo "`pwd`/scripts/python-lint.py --git-precommit" >> .git/hooks/pre-commit
```

Comment:
1. Anyone who can change config_check.py now can screw with everyone's git hooks. Have you considered copying the script?
2. `python-lint.py` is pretty slow, but I guess it's probably OK.
3. `--git-precommit` doesn't work anymore with the outstanding PR to update the linters.